library/src/scala/quoted/Quotes.scala

```diff
@@ -464,6 +464,13 @@ trait Quotes { self: runtime.QuoteUnpickler & runtime.QuoteMatching =>
 
     /** Methods of the module object `val ClassDef` */
     trait ClassDefModule { this: ClassDef.type =>
+      /** Create a class definition tree
+       *
+       *  @param cls The class symbol
```

Comment:
```suggestion
       *  @param cls The class symbol created using `Symbol.newClass`
```

Reply:
Not necessarily, it can be the symbol of a class for which we are transforming the tree.

Reply:
OK, then the comment could just mention that Symbol.newClass should be used to create a new class symbol. Also if we're transforming a tree I think most of the time we should use ClassDef.copy instead of ClassDef.apply.

Reply:
Done